test-files/import-errors.ts

```
@@ -0,0 +1,47 @@
+  return lodash.map([1, 2, 3], x => x * 2)
+}
+
+// Export without proper naming
+export const a = 'value'
+export const b = 42
+
+// Missing default export when file should have one
+export { badFunction }
+
+// Exporting unused function
+export function unusedExportedFunction() {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)